packages.qmd

```diff
@@ -0,0 +1,68 @@
+---
+title: "R Packages"
+sidebar: false
+page-layout: article
+toc: false
+title-block-banner: false
+---
+
+```{r}
+# This function was used to create the default hex logos for packages that don't yet have a custom hex
+
+# p_family <- "Aller_Rg"
+# main_colour <- NHSRtheme::get_nhs_colours()["DarkBlue"]
+# 
+# get_default_hex_sticker <- function(packagename){
+#   sticker <- hexSticker::sticker(
+#     subplot = ggplot2::ggplot() +
+#       hexSticker::theme_transparent(),
+#     package = packagename,
+#     h_color = main_colour,
+#     h_fill = "#ffffff",
+#     s_x = 1,
+#     s_y = 1,
+#     s_height = 0.8,
+#     s_width = 0.8,
+#     asp = 1,
+#     url = "",
+#     u_angle = 30,
+#     u_x = 1.08,
+#     u_y = 0.1,
+#     u_size = 4.85,
+#     dpi = 500,
+#     p_x = 1.0,
+#     p_y = 1,
+#     p_size = 22,  
+#     p_color = main_colour,
+#     p_family = p_family,
+#     u_color = main_colour,
+#     u_family = p_family,
+#     filename = glue::glue("inst/images/{packagename}logo.png")
+#   )
+#   
+#   return(sticker)
+# }
+# 
+# get_default_hex_sticker("NHSRpostcodetools")
+# get_default_hex_sticker("NHSRwaitinglist")
+# get_default_hex_sticker("NHSRpopulation")
+```
+
+The NHS-R community has created and are developing R packages to support analytics in health and care. 
+
+Contributions are very welcome. We appreciate code, documentation, bug fixes and designs for hexes. Please feel free to tackle any of the issues in the package repos or create a new issue. If you have other code or ideas for new packages, please let us know.
```

Comment:
Could repos be written in full for easier reference? Might be useful to put GitHub repositories too to be absolutely clear.